app/Locale/English.php

```diff
@@ -414,11 +414,17 @@ public function get_locale(): array
 			'UPLOAD_IMPORT' => 'Import',
 			'UPLOAD_IMPORT_SERVER' => 'Importing from server',
 			'UPLOAD_IMPORT_SERVER_FOLD' => 'Folder empty or no readable files to process. Please take a look at the log (Settings -> Show Log) for further details.',
-			'UPLOAD_IMPORT_SERVER_INSTR' => 'This action will import all photos, folders and sub-folders which are located in the following directory.',
+			'UPLOAD_IMPORT_SERVER_INSTR' => 'Import all photos, folders and sub-folders located in the folder with the following absolute path (on server):',
 			'UPLOAD_ABSOLUTE_PATH' => 'Absolute path to directory',
 			'UPLOAD_IMPORT_SERVER_EMPT' => 'Could not start import because the folder was empty!',
 			'UPLOAD_IMPORT_DELETE_ORIGINALS' => 'Delete originals',
 			'UPLOAD_IMPORT_DELETE_ORIGINALS_EXPL' => 'Original files will be deleted after the import when possible.',
+			'UPLOAD_IMPORT_VIA_SYMLINK' => 'Symbolic links',
+			'UPLOAD_IMPORT_VIA_SYMLINK_EXPL' => 'Import files using symbolic links to originals.',
+			'UPLOAD_IMPORT_SKIP_DUPLICATES' => 'Skip duplicates',
+			'UPLOAD_IMPORT_SKIP_DUPLICATES_EXPL' => 'Existing media files are skipped.',
+			'UPLOAD_IMPORT_RESYNC_METADATA' => 'Re-sync metadata',
+			'UPLOAD_IMPORT_RESYNC_METADATA_EXPL' => 'Update metadata of existing media files.',
```

Comment:
This will need to be spread to all the other languages as well before merging...

Reply:
Of course, but it's tedious and only worth it if you see merit in this PR. Besides, what is the standard procedure? Just copy & paste the English template and hope for some native speakers or put google translate to the test?

Reply:
This: "Just copy & paste the English template and hope for some native speaker" :)

Reply:
I so wish we could automate it somehow. Meaning that we would only need to add new strings to `English` and it would somehow auto-populate through the rest of them...